doc/nrf/gs_ins_fork.rst

```diff
@@ -3,58 +3,83 @@
 #. Open a GIT bash or terminal window in the ``ncs`` folder.
-#. Clone your forks by entering the following commands, where *<username>* must be replaced with your GitHub user name:
+#. For each of the four repository folders, complete the following steps, replacing *Folder* with the respective folder name (``nrf``, ``zephyr``, ``mcuboot``, or ``nrfxlib``):
 
-   .. code-block:: console
+   a. Enter the respective folder:
 
-      git clone https://github.com/<username>/fw-nrfconnect-zephyr.git zephyr
+      .. parsed-literal::
+         :class: highlight
 
-      git clone https://github.com/<username>/fw-nrfconnect-mcuboot.git mcuboot
+         cd *Folder*
+   #. Add the fork that you created as "default" remote with the name ``origin``.
+      To do so, enter one of the following commands, replacing *Username* with your GitHub user name:
 
-      git clone https://github.com/<username>/fw-nrfconnect-nrf.git nrf
+      For nrf:
+         .. parsed-literal::
+            :class: highlight
 
-      git clone https://github.com/<username>/nrfxlib.git nrfxlib
+            git remote add origin https\://github.com/*Username*/fw-nrfconnect-nrf.git
 
-#. To link your local repositories to the |NCS| repositories, add remotes that point to the original repositories:
+      For zephyr:
+         .. parsed-literal::
+            :class: highlight
 
-   .. code-block:: console
+            git remote add origin https\://github.com/*Username*/fw-nrfconnect-zephyr.git
 
-      cd zephyr
-      git remote add ncs https://github.com/NordicPlayground/fw-nrfconnect-zephyr.git
+      For mcuboot:
+         .. parsed-literal::
+            :class: highlight
 
-      cd ../mcuboot
-      git remote add ncs https://github.com/NordicPlayground/fw-nrfconnect-mcuboot.git
+            git remote add origin https\://github.com/*Username*/fw-nrfconnect-mcuboot.git
 
-      cd ../nrf
-      git remote add ncs https://github.com/NordicPlayground/fw-nrfconnect-nrf.git
+      For nrfxlib:
+         .. parsed-literal::
+            :class: highlight
 
-      cd ../nrfxlib
-      git remote add ncs https://github.com/NordicPlayground/nrfxlib.git
+            git remote add origin https\://github.com/*Username*/nrfxlib.git
+   #. Optionally, for ``zephyr`` and ``mcuboot``, add remotes to the upstream repositories:
 
-#. Optionally, add remotes to the upstream repositories for Zephyr and Mcuboot:
+      For zephyr:
+         .. parsed-literal::
+            :class: highlight
 
-   .. code-block:: console
+            git remote add upstream https\://github.com/zephyrproject-rtos/zephyr.git
 
-      cd ../zephyr
-      git remote add upstream https://github.com/zephyrproject-rtos/zephyr.git
+      For mcuboot:
+         .. parsed-literal::
+            :class: highlight
 
-      cd ../mcuboot
-      git remote add upstream https://github.com/runtimeco/mcuboot.git
+            git remote add upstream https\://github.com/runtimeco/mcuboot.git
```

Comment:
The URL has changed, it was moved into another GitHub user since the acquisition of Runtime by Juul:

https://github.com/JuulLabs-OSS/mcuboot.git